package.json

```diff
@@ -154,6 +154,56 @@
                         "markdownDescription": "Selection of rules used to scan your code with Code Analyzer.\n\nSelect rules using their name, engine name, severity level, tag, or a combination. Use commas for unions (such as \"Security,Performance\") and colons for intersections (such as \"pmd:Security\" or \"eslint:3\").\n\nThis setting is equivalent to the `--rule-selector` flag of the CLI commands. See [examples](https://developer.salesforce.com/docs/atlas.en-us.sfdx_cli_reference.meta/sfdx_cli_reference/cli_reference_code-analyzer_commands_unified.htm#cli_reference_code-analyzer_rules_unified)."
                     }
                 }
+            },
+            {
+                "title": "Diagnostic Levels",
+                "properties": {
+                    "codeAnalyzer.severity 1": {
+                        "type": "string",
+                        "enum": [
+                            "Error",
+                            "Warning"
```

Comment:
Hi @namrata111f 

Correct me if I'm wrong but there is no more 'Info' and 'None' options in this PR.
Did anything changed since our last dicussion ?